buildscripts/scripts/ffmpeg.sh

```diff
@@ -24,7 +24,7 @@ cpuflags=
 
 ../configure \
 	--target-os=android --enable-cross-compile --cross-prefix=$ndk_triple- --cc=$CC \
-	--arch=${ndk_triple%%-*} --cpu=$cpu --pkg-config=pkg-config \
+	--arch=${ndk_triple%%-*} --cpu=$cpu --pkg-config=pkg-config --nm="llvm-nm -g" \
```

Comment:
it doesn't look like the `-g` should be here, that needs to be up to the build script